### DISCUSSION TITLE
What's the criteria for featuring downloads in pureata.info?

### DISCUSSION THREAD

**Discussion by porres** [pure-data/pddp#42] [General]
I'm not sure what qualifies for a 'featured download' in puredata.info, but it seems natural to me it'd refer to some hot new release, that can be one of different categories under 'downloads' (a 'library', a 'distribution', etc.).

It is my understanding that we should not let old and outdated information up there though.

**Comment by porres** [comment]
Also, puredata.info doesn't seem to be the place where people are getting news on the hot releases. These are announced on facebook, the pd-list (by the way, new pd-l2ork was announced on both and probably more places, I only follow more closely these channels) and lots of other places (forum, discord channel, telegram group, etc)...

One way or another, puredata.info is still the main promoted online resource as a central hub of information. I think we need to work on it and try to give an overview of what's going on in terms of new projects, even though we don't have project owners going to puredata.info to show off their stuff. But we also should encourage people to collaborate with the site.

It seems the site is also not the best current solution for most people and they just get discouraged to join. A parallel discussion here is to think of an alternative as in https://github.com/pure-data/pddp/issues/35

**Comment by pd-l2ork** [comment]
As I indicated in my replies to other similar tickets, before we edit the featured list, if there is a planned debate on this topic (this ticket is suggested as serving this purpose), then we need to first debate it, then edit it. This would at the very least require reverting the featured list to its prior state before moving forward and thereby undoing #33 

Pd-L2Ork, based on its age, is as far as I can tell one of the longest if not the longest maintained pd variants, of which Purr-Data is, by definition a sub-variant. Purr-Data even used the Pd-L2Ork name 2016-2020, calling itself Pd-L2Ork 2.x. It continues to use this dual name despite my recent requests to cease doing so to minimize confusion.

**Comment by porres** [comment]
The site was edited before much discussion, but it was made in good will and represented an improvement.

One concern of mine here and in previous issues is to not feature outdated information. For instance, we had the Pd-l2ork page featured before, so in https://github.com/pure-data/pddp/issues/38 I hoped we could update it (if not, remove it for a more up to date and less ambiguous entry); having updated information being the first option and ideal goal (I also promoted up to date information on the front page).

I think @umlaeute is right in assuming that "Purr Data" is the 'top of mind' feature. And regarding the prior state of things, the old entry was Pd-L2Ork, but it referred to Purr Data as the "current release". In practice, it did point and refer to Purr Data. So, in a sense, it is not crazy to consider featuring Purr Data is the prior state of things.

To make matters yet more confusing, the Pd-l2ork entry pointed to purr data, but purr data has this separate page, now featured and with out of date information.

So now I have the same problem. The featured Purr entry is out of date. I'd much rather have Pd-L2Ork's entry back in then as an alternative. 

Regarding the topic of the discussion, I can ask, what do we expect to see in featured downloads? Presumable "top of mind" projects?

I have an idea that it could be a place for featuring and highlighting the main recent update/release. This would make sense if the download page had several active projects providing updates all the time. Not really what's going on...

Nonetheless, under that perspective, having Pd-L2Ork emerge now as an independent project seems like big news and something that would apply for a featured download under this idea.

So yeah, I'd like to suggest to @umlaeute to just put l2ork back as a feature and we can keep discussing here what we expect for featured downloads.

**Comment by porres** [comment]
We have separate discussion here too (which is fine), regarding the relationship between pd-l2ork and purr data. 

> Pd-L2Ork, based on its age, is as far as I can tell one of the longest if not the longest maintained pd variants, of which Purr-Data is, by definition a sub-variant. Purr-Data even used the Pd-L2Ork name 2016-2020, calling itself Pd-L2Ork 2.x. It continues to use this dual name despite my recent requests to cease doing so to minimize confusion.

I see, in fact naming the software the same is a big issue. I have both here installed and have to differentiate them by the icon! But then, the last release I see is https://github.com/agraef/purr-data/releases from august. If there we newer ones, it would be really really worse...

Yeah, it seems both of the project still need to properly officialize they're unmerging. Actually, Purr Data needs to do that! But then, I wouldn't like to see here a discussion on which one gets to prevail as the 'real one', the 'main one'... 

Now, it is also important to notice that the download pages on puredata.info, originally, offered a place to upload files that people could then download. These pages would then get updates at every release. Currently, we're using it to refer to external repositories and official webpages. And people just get directed there to see the check for updates and current releases...

In this respect, there can be the suggestion to have a single pure data page for both projects, maybe. And have that page featured. Of course that would mean that the developers of both project were really really aligned and it seems they cannot even well coordinate they getting split into independent projects...

I say this because it would really be weird if we had a discussion that both needed to be featured in the downloads in order to make everyone happy and because both are so important. That seems weird to me. Ok they're separate and independent projects now, but they're still quite close and similar siblings, specially taking in consideration how recently this is. Back in 2009 I can assume Pd-L2Ork was pretty much about the same thing as Pd-extended...

**Comment by pd-l2ork** [comment]
The two projects are already fairly different and things that work in one
do not work in other. In the last year alone, pd-l2ork has had hundreds of
commits. So, no, we should not group them because they are as similar to
each other as they are to vanilla.

Best,

Ico

-- 
Ivica Ico Bukvic, D.M.A.
Director, Creativity + Innovation
Institute for Creativity, Arts, and Technology

Virginia Tech
Creative Technologies in Music
School of Performing Arts – 0141
Blacksburg, VA 24061
(540) 231-6139
***@***.***

ci.icat.vt.edu
l2ork.icat.vt.edu
ico.bukvic.net

**Comment by pd-l2ork** [comment]
I am also not sure what was the improvement in the featured section that
you speak of. In my view this was not an improvement at all because my
input was effectively ignored and the change was made without a consensus
among those whom the decision affects the most. So, if you are looking to
make a change in good will, you should first revert and then update based
on the input and the ensuing consensus. Or, at the very least make a change
that is more inclusive rather than exclusive, meaning having all 4 variants
there. Worse yet, you are now featuring a variant whose page is out of
date, while excluding the one that has actually been updated.

Best,

Ico

-- 
Ivica Ico Bukvic, D.M.A.
Director, Creativity + Innovation
Institute for Creativity, Arts, and Technology

Virginia Tech
Creative Technologies in Music
School of Performing Arts – 0141
Blacksburg, VA 24061
(540) 231-6139
***@***.***

ci.icat.vt.edu
l2ork.icat.vt.edu
ico.bukvic.net

**Comment by porres** [comment]
I didn't specifically said there was an improvement on the featured section alone. Edits to your outdated entry represented an improvement at the moment and was in good will. Other improvements have been made to the front page. I also already agreed the way things are now are bad, I said I wasn't happy with it before you joined the discussion. And I also already said I'd rather have your updated entry instead.

**Comment by porres** [comment]
> So yeah, I'd like to suggest to @umlaeute to just put l2ork back as a feature and we can keep discussing here what we expect for featured downloads.

ping @umlaeute again

btw, the actual discussion is not really happening here... 

[edit]: pd-l2ork is back being featured!

**Comment by porres** [comment]
Closing this as it seems it died on its own ages ago and didn't go anywhere and I'm not sure where it was supposed to go. For the record, I agree with my original idea, it must refer to some hot new release, and taking into consideration how things go these days, it should be a fork... and, well, anyway, "hot new" is arbitrary, of course...
